lib/GithubUtils.js

```diff
@@ -65,22 +65,84 @@ export default class GithubUtils {
             const compareURLRegex = new RegExp(`${EXPENSIFY_CASH_URL}/compare/${versionRegex.source}\\.\\.\\.${versionRegex.source}`, 'g');
             const comparisonURL = issue.body.match(compareURLRegex)[0];
 
-            const checklistItemRegex = new RegExp(`- \\[[ x]] (${ISSUE_OR_PULL_REQUEST_REGEX.source})`, 'g');
-
-            const PRListSection = issue.body.match(/pull requests:\*\*\r\n((?:.*\r\n)+)\r\n/)[1];
-            const PRList = [...PRListSection.matchAll(checklistItemRegex)].map(match => match[1]);
-
-            const deployBlockerSection = issue.body.match(/Deploy Blockers:\*\*\r\n((?:.*\r\n)+)/)[1];
-            const deployBlockers = [...deployBlockerSection.matchAll(checklistItemRegex)].map(match => match[1]);
-
             return {
-                title: issue.title, url: issue.url, labels: issue.labels, tag, comparisonURL, PRList, deployBlockers,
+                title: issue.title,
+                url: issue.url,
+                labels: issue.labels,
+                PRList: this.getStagingDeployCashPRList(issue),
+                deployBlockers: this.getStagingDeployCashDeployBlockers(issue),
+                tag,
+                comparisonURL,
             };
         } catch (exception) {
             throw new Error(`Unable to find ${STAGING_DEPLOY_CASH_LABEL} issue with correct data.`);
         }
     }
 
+    /**
+     * Parse the PRList section of the StagingDeployCash issue body.
+     *
+     * @private
+     *
+     * @param {Object} issue
+     * @returns {Array<Object>} - [{url: String, number: Number, isVerified: Boolean}]
+     */
+    getStagingDeployCashPRList(issue) {
+        const PRListSection = issue.body.match(/pull requests:\*\*\r\n((?:.*\r\n)+)\r\n/)[1];
```

Comment:
Should we just standardize on `RegExp`? Any reason to use a regex literal here?

Reply:
Only that this one isn't really being reused. I don't think there's any need to standardize on `RegExp`...I think `RegExp` should be used if you're going to reuse the regex to compose another one, otherwise regex literal is fine.